modules/samples/data-populator/Populator/src/org/wso2/carbon/registry/samples/populator/Main.java

```diff
@@ -69,21 +69,46 @@ public void setCookie(String cookie) {
             int tasks = 10;
             String projectPath = System.getProperty("user.dir");
 
-            addWsdlGar(resourceServiceClient, projectPath);
-            Thread.sleep(1 * 60 * 1000);
-            System.out.println("######## Successfully uploaded sample wsdls ########");
-            addWadlGar(resourceServiceClient, projectPath);
-            Thread.sleep(30 * 1000);
-            System.out.println("######## Successfully uploaded sample wadls ########");
-            addSchemaGar(resourceServiceClient, projectPath);
-            Thread.sleep(30 * 1000);
-            System.out.println("######## Successfully uploaded sample schemas ########");
-            addSwaggerGar(resourceServiceClient, projectPath);
-            Thread.sleep(30 * 1000);
-            System.out.println("######## Successfully uploaded sample swagger docs ########");
-            addPolicyGar(resourceServiceClient, projectPath);
-            Thread.sleep(30 * 1000);
-            System.out.println("######## Successfully uploaded sample policies ########");
+            try {
+                addWsdlGar(resourceServiceClient, projectPath);
+                Thread.sleep(1 * 60 * 1000);
```

Comment:
This is fixed with PR : https://github.com/wso2/product-greg/pull/300/
only 10 seconds sleep is added to wsdl upload , for resources to be indexed.

Thanks